tests/appsec/test_blocking_addresses.py

```diff
@@ -722,3 +723,60 @@ def test_non_blocking(self):
     def test_blocking_before(self):
         """Test that blocked requests are blocked before being processed"""
         # TODO
+
+
+@scenarios.appsec_blocking
+@coverage.good
+@released(
```

Comment:
Could you use the new manifest files for this please ?

Reply:
sure